bats/fb-katello-content.bats

```diff
@@ -89,6 +106,21 @@ setup() {
     --path="/tmp/stbenjam-dummy-0.2.0.tar.gz" | grep -q "Successfully uploaded"
 }
 
+@test "upload ostree_ref" {
+  if ! tIsRHEL 8; then
+    skip "OSTree content is only applicable on EL 8 systems"
+  fi
+
+  tSkipIfOlderThan43
+
+  wget --no-parent -r https://fixtures.pulpproject.org/ostree/small/
```

Comment:
This finds at least one 404, so I think it's failing the whole test even though the files are being downloaded:

```
[root@centos8-katello-nightly bats]#  wget --no-parent -r https://fixtures.pulpproject.org/ostree/small/
--2022-01-10 20:50:09--  https://fixtures.pulpproject.org/ostree/small/
Resolving fixtures.pulpproject.org (fixtures.pulpproject.org)... 35.163.140.6, 34.209.16.239
Connecting to fixtures.pulpproject.org (fixtures.pulpproject.org)|35.163.140.6|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘fixtures.pulpproject.org/ostree/small/index.html’

fixtures.pulpproject.org/ostree/small/index.html                                  [ <=>                                                                                                                                                                                             ]     959  --.-KB/s    in 0s      

2022-01-10 20:50:10 (17.4 MB/s) - ‘fixtures.pulpproject.org/ostree/small/index.html’ saved [959]

Loading robots.txt; please ignore errors.
--2022-01-10 20:50:10--  https://fixtures.pulpproject.org/robots.txt
Reusing existing connection to fixtures.pulpproject.org:443.
HTTP request sent, awaiting response... 404 Not Found
2022-01-10 20:50:10 ERROR 404: Not Found.
```

Reply:
I wouldn't think so because it literally states:
> Loading robots.txt; please ignore errors.

It's just following https://en.wikipedia.org/wiki/Robots_exclusion_standard

Reply:
Ah, I think `tFileExists` is failing because the file in question is a directory: https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

`-f` means "True if file exists and is a regular file. "
`-d` means "True if file exists and is a directory. "

I'll test this.

Reply:
That was it! I just added a `tDirectoryExists` command and changed the `-f` to a `-d`.

@jjeffers ^

Reply:
Where can I pull in that change?

Reply:
@jjeffers just copy/paste `tFileExists` into a new `tDirectoryExists` and change the `-f` to a `-d`.

Reply:
Simple enough!